tools/circt-lsp-server/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+set(LLVM_OPTIONAL_SOURCES
+  null.cpp
```

Comment:
Just curious, is the optional source here needed?

Reply:
Not sure, just copied the `mlir-lsp-server` CMakeLists... I say we keep it unless someone can guarantee removing it doesn't break something :)